cpp/models/ode_secirvvs/parameters_io.h

```diff
@@ -1197,7 +1197,7 @@ IOResult<void> export_input_data_county_timeseries(std::vector<Model>&& model, c
                                                    double scaling_factor_icu, int num_days, bool set_vaccination_data)
 {
     if (set_vaccination_data) {
-        BOOST_OUTCOME_TRY(details::set_vaccination_data(model, path_join(data_dir, "all_county_ageinf_vacc_ma7.json"),
+        BOOST_OUTCOME_TRY(details::set_vaccination_data(model, path_join(data_dir, "vacc_county_ageinf_ma7.json"),
```

Comment:
If you rename this, maybe there is a better thing than ma7?

Reply:
What do you suggest?

Reply:
I would need to look at the data, what does ma7 currently stand for?

Reply:
moving average for 7 days

Reply:
vacc_county_ageinf_mov_avg_7

Reply:
looks better for me. But since this functionality is used for most data downloads this first needs to be discussed.